pineapple-core/src/main/java/sh/miles/pineapple/command/CommandRegistry.java

```diff
@@ -19,8 +15,6 @@
 public final class CommandRegistry {
```

Comment:
I Honestly think we can just get rid of this class its not worth to have it for a shorthand. If anything we can literally just inline a `registerCommand(Consumer<ReloadableRegistrarEvent<Command>>)` method to pineapple to make feel easier.

https://jd.papermc.io/paper/1.21.4/io/papermc/paper/plugin/lifecycle/event/types/LifecycleEvents.html#COMMANDS

This is technically a break, but while we're walking around with a sledge hammer busting shit up anyways might as well make this make sense.